<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="GUDU-PC" name="com.employeeapi.testCases.TC002_Get_Single_Employee_Record" tests="7" failures="0" timestamp="27 May 2021 04:18:38 GMT" time="0.004" errors="1">
  <testcase name="checkContentLenght" time="0.001" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record">
    <error type="java.lang.NumberFormatException" message="null">
      <![CDATA[java.lang.NumberFormatException: null
at java.lang.Integer.parseInt(Integer.java:542)
at java.lang.Integer.parseInt(Integer.java:615)
at com.employeeapi.testCases.TC002_Get_Single_Employee_Record.checkContentLenght(TC002_Get_Single_Employee_Record.java:93)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- checkContentLenght -->
  <testcase name="checkContentType" time="0.001" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record"/>
  <testcase name="checkResposeBody" time="0.001" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record"/>
  <testcase name="checkserverType" time="0.000" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record"/>
  <testcase name="checkResponseTime" time="0.000" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record"/>
  <testcase name="checkStatusCode" time="0.001" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record"/>
  <testcase name="checkstatusLine" time="0.000" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record"/>
</testsuite> <!-- com.employeeapi.testCases.TC002_Get_Single_Employee_Record -->
